sormas-backend/src/main/java/de/symeda/sormas/backend/contact/ContactFacadeEjb.java

```diff
@@ -2391,6 +2391,17 @@ public User getRandomRegionContactResponsible(Region region) {
 		return userService.getRandomRegionUser(region, UserRight.CONTACT_RESPONSIBLE);
 	}
 
+	@Override
+	@RightsAllowed(UserRight._PERSON_EDIT)
+	public void startHomeAddressAdoption(ContactDto dto) {
```

Comment:
From my point of view this method is too specific. It just updates the address of one person to be the same as the address of another person.
I'd either do this directly in the frontend OR instead create a method in PersonFacade, e.g. PersonFacade.copyAddress(PersonReferenceDto source, PersonReferenceDto target).